NEXTCLOUD.cmake

```diff
@@ -1,37 +1,44 @@
-set( APPLICATION_NAME       "Nextcloud" )
-set( APPLICATION_SHORTNAME  "Nextcloud" )
-set( APPLICATION_EXECUTABLE "nextcloud" )
-set( APPLICATION_DOMAIN     "nextcloud.com" )
-set( APPLICATION_VENDOR     "Nextcloud GmbH" )
+mark_as_advanced( FORCE APPLICATION_NAME APPLICATION_SHORTNAME APPLICATION_EXECUTABLE APPLICATION_DOMAIN APPLICATION_VENDOR APPLICATION_ICON_NAME APPLICATION_REV_DOMAIN LINUX_PACKAGE_SHORTNAME THEME_CLASS )
```

Comment:
I think FORCE is not necessary. Also I would find it more convenient to place that `mark_as_advanced` statement at the end of the file.

Reply:
even if you are probably true, setting `FORCE` has the advantage to ensure we get the desired state whatever is in the cmake cache